client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.java

```diff
@@ -95,6 +94,24 @@
     public void changeBrand(Brand brand) {
```

Comment:
Is this always called (even without "changing brand")? If not, the href won't get set the first time.